Gemfile

```diff
@@ -19,8 +19,8 @@ gem 'bowndler', github: 'moneyadviceservice/bowndler'
 # is the same as the Gem version.
 gem 'dough-ruby',
     git: 'https://github.com/moneyadviceservice/dough.git',
-    branch: 'master',
-    ref: '238a329'
+    branch: '11363_Sticky-banner_Coronavirus-info',
+    ref: '1e526e6'
```

Comment:
Did you mean to change this back to the master branch?

Reply:
@andreferreiradlw Sorry, it might have been unclear before, but the Dough branch isn't merged yet. I think this reference should remain in place for testing. When that's approved the Dough branch can be merged and this reference updated.